cmd/consul-dataplane/main.go

```diff
@@ -113,7 +113,7 @@ func init() {
 	flag.StringVar(&promKeyFile, "telemetry-prom-key-file", "", "The path to the client private key used to serve Prometheus metrics.")
 	flag.StringVar(&promCertFile, "telemetry-prom-cert-file", "", "The path to the client certificate used to serve Prometheus metrics.")
 	flag.StringVar(&promServiceMetricsURL, "telemetry-prom-service-metrics-url", "", "Prometheus metrics at this URL are scraped and included in Consul Dataplane's main Prometheus metrics.")
-	flag.StringVar(&promScrapePath, "telemetry-prom-scrape-path", "", "The URL path where Envoy serves Prometheus metrics.")
+	flag.StringVar(&promScrapePath, "telemetry-prom-scrape-path", "/metrics", "The URL path where Envoy serves Prometheus metrics.")
```

Comment:
I chose /metrics to match the [k8s docs](https://developer.hashicorp.com/consul/docs/k8s/connect/observability/metrics)?